doc/tutorials/using_cat_on_up2.rst

```diff
@@ -0,0 +1,164 @@
+.. _using_cat_up2:
+
+Using CAT on UP2
+################
+
+The UP Squared board `(UP2) <https://up-board.org/upsquared/specifications/>`_
+is an x86 maker board based on the Intel Apollo Lake platform and supports
+Cache Allocation Technology (CAT). With this feature, the usage of cache can be
+restricted to each VM. ACRN hypervisor can do that by giving each VM a cache mask,
+so that the VM will not evict the masked cache lines, as shown in :numref:
+`cache-mask-vm`:
+
+.. figure:: images/using_cat_up2.png
+   :align: center
+   :name: cache-mask-vm
+
+   Cache usage with Cache Mask
+
+
+CAT Support on ACRN
+*******************
+
+As described at `Intel |reg| 64 and IA-32 Architectures Software Developer’s Manual
```

Comment:
My apologies @taoyuhong , it turns out my suggestion to change the (R) sign into `|reg|` wasn't a good one... Sphinx/docutils do not support such substitutions... see https://github.com/sphinx-doc/sphinx/issues/2793 :(

Reply:
OK

Reply:
We can use 'Intel (R)', thank you Geoff